### DISCUSSION TITLE
Decouple Channels from Daphne

### DISCUSSION THREAD

**Discussion by carltongibson** [django/channels#1643] [Ideas] [answered]
* Folks want use Channels without Daphne. 
* Other folks want a simple story to get up and running, which doesn't need you to be a Python packaging expert. 

Historically we've favoured the 2nd of those. **But** keen to think if we can solve the problem.

Requirements: 

1. The code and packaging changes. 
2. The tutorial/docs steps spelt out clearly. 

References: 

* #1353 was looking at this. Noting here because it got closed renaming the `main` branch. 
* #1639 and https://github.com/django/daphne/pull/359
* ... plenty of others ... (gather and add)

**Comment by carltongibson** [comment]
Looking to resolve this in #1897.

**Comment by carltongibson** [comment, marked as answer]
This is done and available in 4.0.0b1. 

See #1898

**Reply by Archmonger** [reply]
Since this feature was completed a while ago, it should probably be [unpinned](https://github.com/django/channels/discussions).

**Reply by carltongibson** [reply]
Thanks @Archmonger 😅 Yep!
